app/views/conferences/_booths.html.haml

```diff
@@ -0,0 +1,24 @@
+= content_for :splash_nav do
+  %li
+    %a.smoothscroll{ href: '#booths' } Booths
+
+.container
+  .row
+    .col-md-12.text-center
+      %h2 Booths
+  - @conference.booths.confirmed.each_slice(3).with_index do |slice, i|
```

Comment:
Variables with names that don't convey a meaning is not the right way to write code. You are obviously not able to write the code properly using slice, what about you try without it?